.changes/1.4.0-a1.md

```diff
@@ -0,0 +1,75 @@
+## dbt-core 1.4.0-a1 - September 28, 2022
+
+### Breaking Changes
+
+- Renaming Metric Spec Attributes ([#5774](https://github.com/dbt-labs/dbt-core/issues/5774), [#5775](https://github.com/dbt-labs/dbt-core/pull/5775))
+
```

Comment:
I think all of these changes belong in 1.3 and not here.  Related issue: #5074

Reply:
@emmyoop I updated this to look like what we did last time. I think this look right but would you take another look to double check please?